### ISSUE TITLE
Why so ugly?

### ISSUE THREAD

**Issue by andersonwinkler** [UTRGV-SMSS/Thesis-Template#31]
I'm helping one of our PhD students to prepare their thesis materials. It's embarrassing for me and for the student that our University template is so ugly.

Why are the chapter headings with the same font size as the body of the text? Why capitalization there? If the chapter title is long, it's hard to read. Why is the formatting of each of the chapters, sections, and subsections so poorly discriminated? Why no headings on each page with the chapter/section names to help locate where we are reading? Why Times New Roman when Donald Knuth went through great lengths to create a font that would look really nice for mathematical formulas in LaTeX? We can't even replace that. Why not more contemporary fonts that allow classical ligatures?

It seems who originally prepared our University template was used to typewriters or the very first versions of Microsoft Word. We are in 2024 already. Our students are using sophisticated software and can apply beautiful formatting that would very easily make reading the document a pleasure.

Also, the thesis is a lifetime achievement for the students. In a way, it is a piece of art. Why are we forcing them to produce an ugly product that they will be ashamed of showing others?

Even LaTeX default `report` class is far (**far!**) nicer than our template.

To whom do we have to appeal to have the template changed or, at least, how can we give the students freedom to choose a better formatting or make much needed improvements to the current one, and that would be more in line with all the effort that they put into the content?

Thanks! (actually, no, not thanks, since the template is really abysmal)

Anderson

**Comment by andersonwinkler** [comment]
Thanks @ggarza. I appreciate your efforts in maintaining a LaTeX-compatible template, a daunting task on its own right.

After writing here I emailed the Graduate College to ask what the process to change the template would be. Let's see what they say.

Even if we settled for the default style in Word or LaTeX, we'd still be better off. Not only these look better, but also these aren't as demotivating for the student as the current one.

If you permit, I'll email you when I hear back from them. Maybe we could have a small committee working on a simple but nicer format that would look somewhat similar across Word, LaTeX, and LibreOffice, yet be less boring than the current one.

Thanks!

Anderson
